### ISSUE TITLE
PHP User Error (256): Trying to load a forbidden class name "0" in /glpi/css/inc/autoload.function.php at line 280

### ISSUE THREAD

**Issue by carlosbayerl** [glpi-project/glpi#13306]
### Code of Conduct

- [ ] I agree to follow this project's Code of Conduct

### Is there an existing issue for this?

- [ ] I have searched the existing issues

### Version

9.5.11

### Bug description

Dear, when checking an already opened ticket, which contained several follow-ups, I got the error "PHP User Error (256): Trying to load a forbidden class name "0" in /glpi/css/inc/autoload.function.php at line 280". How do I fix this and get back to viewing the call normally?

### Relevant log output

```shell
[2022-11-16 09:49:29] glpiphplog.ERROR:   *** PHP User Error (256): Trying to load a forbidden class name "0" in /glpi/css/inc/autoload.function.php at line 280
  Backtrace :
  inc/autoload.function.php:280                      trigger_error()
  :                                                  glpi_autoload()
  :                                                  spl_autoload_call()
  inc/commonitilobject.class.php:8418                class_exists()
  inc/ticket.class.php:704                           CommonITILObject->getLinkedItems()
  inc/commonitilobject.class.php:7960                Ticket->getTabNameForItem()
  inc/ticket.class.php:818                           CommonITILObject->defineDefaultObjectTabs()
  inc/commonglpi.class.php:294                       Ticket->defineTabs()
  inc/commonglpi.class.php:859                       CommonGLPI->defineAllTabs()
  inc/commonglpi.class.php:1231                      CommonGLPI->showTabsContent()
  front/ticket.form.php:221                          CommonGLPI->display()
```

**Comment by cedric-anne** [comment]
Hi,

You probably have an entry with `itemtype=0` in `glpi_items_tickets` table. This is probably due to a buggy plugin, or a missing plugin migration.
If you have suche entries, delete them, they are not valid.

**Comment by cedric-anne** [comment]
Fixed in 10.0 by #9668.
